github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -169,12 +169,13 @@ presubmits:
     annotations:
       fork-per-release: "true"
       k8s.v1.cni.cncf.io/networks: multus-cni-ns/sriov-passthrough-cni
-    always_run: false
-    optional: true
+    always_run: true
+    optional: false
```

Comment:
I think we cannot make it required yet.
The lane itself is flaky, we need to start monitoring it and then move it to non-optional.

Reply:
for more info
https://github.com/kubevirt/project-infra/pull/680#pullrequestreview-518660540